tasks/redhat.yml

```diff
@@ -0,0 +1,58 @@
+---
+# tasks file for roles/ice
+
+- name: zeroc ice | setup repository
+  become: true
+  get_url:
+    dest: /etc/yum.repos.d/zeroc-ice-el7.repo
+    url: "https://zeroc.com/download/Ice/3.6/el7/zeroc-ice3.6.repo"
+
+- name: zeroc ice | install runtime packages
+  become: true
+  yum:
+    name: ice-all-runtime
+    state: present
+
+- name: zeroc ice | install devel packages
+  become: true
+  yum:
+    name: ice-all-devel
+    state: present
+  when: ice_install_devel
+
+- name: zeroc ice | install ice pip dependencies
+  become: true
+  yum:
+    name:
+      - gcc
```

Comment:
We have a different set of dependencies on omero-install (some probably from before we made the switch to https://github.com/ome/zeroc-ice-ubuntu1804...
Something to check. I can reduce to match that set and see what we got

Reply:
This is unchanged from the previous version of the role, it's just that it's been moved to a separate file